_episodes_rmd/03-func-R.Rmd

```diff
@@ -310,20 +310,23 @@ The descriptive tags (preceded by an `@` symbol) hold the most important details
 output. An accurate description of the in- and output data (types) helps to 
 successfully apply a function and integrate it into larger analysis pipelines.
 
-```{r challenge-more-advanced-function-analyze, eval=FALSE, include=FALSE}
```

Comment:
`eval=FALSE` AND `include=FALSE` without problems so far, mean that the following code can be wrapped into the Carpentries-style challenge block, right? I didn't notice any problems in the rest of the material, but I could be wrong.

Reply:
@katrinleinweber Hi! I've used the `<details>` html tag wrapped in R code chunks to hide solutions, but still let knitr run and output the results. See an [here for an example](https://dda-rcourse.lwjohnst.com/course/data-management/#exercise-filtering-and-logic-arranging-and-selecting) and [here for the source](https://gitlab.com/lwjohnst/dda-rcourse/blob/master/content/course/data-management.Rmarkdown#L167). Just a suggestion! :relaxed:

Reply:
Thanks for the hint! I'll stick to the Carpentries-style challenge blocks though. I think the `eval=FALSE` was [an artifact from a year-old decision](https://github.com/TIBHannover/FAIR-R/commit/667666d3233a6b79180fa64b175459ac88627fbf).